### ISSUE TITLE
Satchel Auto-Collect Treats Stone, Andesite, Granite, Diorite Identically

### ISSUE THREAD

**Issue by jaallman** [CoFH/Feedback#1167]
What the title says: When I enable satchels to auto-collect, and whitelist stone, the satchel also collects andesite, diorite, and granite; when I whitelist stone, those three are also excluded from being collected by the satchel. The same behavior occurs when whitelisting or blacklisting any one of the other three: all three along with stone are either collected or excluded. Other items--dirt, gravel, ores, etc.--are successfully excluded or included by the satchels.

Basically, the satchel treats stone, andesite, granite and diorite as though they are the same thing. 

I tested servos, filters, and retrievers, and these do NOT treat the four stones identically. (That is, whitelisting one does not whitelist the other three.) I tested it with only the CoFH suite of mods, and it still happens, so it doesn't appear to involve other mods.

This is not a game-breaking issue, but I thought I should mention it.

Thanks!

**Comment by KingLemming** [comment]
So. There are options for metadata sensitivity and such. You have to use them, as they are the same block ID.

This is intentional and correct behavior.
